pkg/daemon/daemon.go

```diff
@@ -515,3 +533,24 @@ func tryCreateUdevRule() error {
 	}
 	return nil
 }
+
+func (dn *Daemon) rebootDisabled() bool {
+
+	deploymentRes := schema.GroupVersionResource{
+		Group:    "sriovnetwork.openshift.io",
+		Version:  "v1",
+		Resource: "sriovoperatorconfigs",
+	}
+	sriovoperatorconfig, err := dn.dynamicclient.Resource(deploymentRes).Namespace("default").Get("default", metav1.GetOptions{})
+	if err != nil {
+		glog.V(2).Infof("rebootDisabled(): Failed to check disableReboot: SriovOperatorConfig not found.")
+		return false
+	}
+	spec := sriovoperatorconfig.Object["spec"].(map[string]interface{})
+	isreboot, ok := spec["disableReboot"]
+	if !ok {
+		glog.V(2).Infof("rebootDisabled(): Failed to check disableReboot: SriovOperatorConfig does not contain disableReboot.")
+		return false
+	}
+	return isreboot.(bool)
```

Comment:
Can you also check that the variable is really a bool (otherwise it will panic)?
res, ok := isreboot.(bool)

Reply:
Done